shared-bindings/busio/I2C.c

```diff
@@ -46,7 +46,7 @@
 //|         lines respectively.
 //|
 //|         .. seealso:: Using this class directly requires careful lock management.
-//|             Instead, use :class:`~adafruit_bus_device.i2c_device.I2CDevice` to
+//|             Instead, use :class:`~adafruit_bus_device.I2CDevice` to
```

Comment:
I think this is not right? It's still inside `i2c_device`.

Reply:
Its just they way the docs are laid out. Initially I tried `~adafruit_bus_device.i2c_device.I2CDevice`, but the rendered docs didn't "auto-link" correctly. You're correct though; these classes do exist in their own module (as they do in the pure python lib also). I didn't pursue fixing this inconsistency because the tilda `~` hides the inconsistency.